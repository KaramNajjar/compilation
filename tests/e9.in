123 321 
obj
<</Butter 5 /nut>>
endobj