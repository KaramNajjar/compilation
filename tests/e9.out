Producing Obj
Producing Body
Producing Dict
Producing KVList
Producing KV
Producing Exp
Finished producing Exp
Finished producing KV
Producing KVList
Producing KV
Producing Exp
Syntax error
